Sources/XcodeGraph/Models/Metadata/TargetMetadata.swift

```diff
@@ -4,29 +4,39 @@ public struct TargetMetadata: Codable, Equatable, Sendable {
     /// Some Tuist features can leverage that information for doing things like filtering.
     public var tags: Set<String>
 
+    /// Whether the target redundant dependencies should be ignored during `tuist inspect redundant-import`
+    public var ignoreRedundantDependencies: Bool
```

Comment:
XcodeGraph models should contain attributes/metadata that's associated with the graph, not with how some Tuist features will behave because it couples the model with the tool that uses the graph. I'd recommend to move this to the `Tuist.swift` model in `tuist/tuist`:

```swift
let tuist = Tuist(inspect: .inspect(redundantDependencies: .redundantDependencies(disableTargets: []))
```